portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
```

Comment:
Maybe change this function's name to "addRandomInterest" and the vars in this function to something to do with "interest".